### PR TITLE
Added recipe for potranslator.

### PR DESCRIPTION
**Pull request by SekouD**
https://pypi.org/project/potranslator/

### CODE REVIEW

**Comment by SekouD** [comment]
#6319

**Comment by SekouD** [comment]
importlib_resources is not the cause of the CI build fail.

I will investigate why the conda-build script complains about hard-coded paths

**Comment by SekouD** [comment]
I just checked the code of all modules in potranslator and I can't seem to find any hard-coded paths.

File: recipes/potranslator/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+    - googletrans
+    - polib
+    - path.py
+    - importlib_resources
+
+test:
+  imports:
+    - potranslator
+
+
```
Comment (marcelotrevisani) [inline recipes/potranslator/meta.yaml]: Please, let just one blank line here

File: recipes/potranslator/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+  entry_points:
+    - potranslator=potranslator.commands:main
+  script: python -m pip install --no-deps --ignore-installed .
+
+requirements:
+  host:
+    - python
+    - pip
+    - setuptools
+    - pytest-runner
```
Comment (marcelotrevisani) [inline recipes/potranslator/meta.yaml]: is the ``pytest-runner`` necessary to build the package?

**Comment by SekouD** [comment]
hi @marcelotrevisani The build fails if pytest-runner is not included.

I will add it back for now.

**Comment by SekouD** [comment]
Now I still have the same error message I had from the beginning:

`
Fixing permissions
Detected hard-coded path in text file python-scripts/potranslator
Traceback (most recent call last):
  File "/home/conda/.ci_support/build_all.py", line 83, in <module>
    build_all(args.recipes_dir, args.arch)
  File "/home/conda/.ci_support/build_all.py", line 73, in build_all
    conda_build.api.build(list(d.keys()), config=config)
  File "/opt/conda/lib/python3.6/site-packages/conda_build/api.py", line 200, in build
    notest=notest, need_source_download=need_source_download, variants=variants)
  File "/opt/conda/lib/python3.6/site-packages/conda_build/build.py", line 2189, in build_tree
    notest=notest,
  File "/opt/conda/lib/python3.6/site-packages/conda_build/build.py", line 1575, in build
    built_package = bundlers[output_d.get('type', 'conda')](output_d, m, env, stats)
  File "/opt/conda/lib/python3.6/site-packages/conda_build/build.py", line 992, in bundle_conda
    exit_on_error=metadata.config.exit_on_verify_error)
  File "/opt/conda/lib/python3.6/site-packages/conda_verify/verify.py", line 20, in verify_package
    package_check = CondaPackageCheck(path_to_package)
  File "/opt/conda/lib/python3.6/site-packages/conda_verify/checks.py", line 41, in __init__
    self.paths_file = self.archive.extractfile('info/paths.json').read()
  File "/opt/conda/lib/python3.6/tarfile.py", line 2074, in extractfile
    tarinfo = self.getmember(member)
  File "/opt/conda/lib/python3.6/tarfile.py", line 1750, in getmember
    raise KeyError("filename %r not found" % name)
KeyError: "filename 'info/paths.json' not found"
Exited with code 1
`

Yet I looked for hard-coded paths in all the modules of potranslator and could not find any.

**Comment by SekouD** [comment]
Hi @marcelotrevisani do you have any pointers regarding the build fail on circleci?

**Comment by marcelotrevisani** [comment]
I will try to test that later in my machine an I will come back to you. The thing is, in your code is trying to see for the file ``info/paths.json`` and it seems that does not exist

**Comment by SekouD** [comment]
Thank you @marcelotrevisani I checked all the code but I can't find any hardcoded path and there is no reference to info/paths.json in my code.

**Comment by SekouD** [comment]
Hi @marcelotrevisani did you have time to check why the build fails on circleci ?

File: recipes/potranslator/LICENSE [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+
+THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS"
+AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE
+DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE
+FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL
+DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR
+SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER
+CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY,
+OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
```
Comment (marcelotrevisani) [inline recipes/potranslator/LICENSE]: Please, delete this file, since the license is already being downloaded and package this part is not necessary

**Comment by marcelotrevisani** [comment]
Please, maybe it was an error in circle ci, please see my comments in your recipe to delete the license which you uploaded. That will trigger the ci again, if you see the same error please ping me again

**Comment by SekouD** [comment]
Thanks @marcelotrevisani After removing the LICENCE file all builds are passing

**Comment by marcelotrevisani** [comment]
Thanks for your effort to put this in conda-forge!

**Pull request merged**